BlazarUI/app/scripts/components/build/Build.jsx

```diff
@@ -12,11 +12,24 @@ import SectionLoader from '../shared/SectionLoader.jsx';
 import Icon from '../shared/Icon.jsx';
 import { BUILD_ICONS } from '../constants';
 import BuildCommits from './BuildCommits.jsx';
+import Button from 'react-bootstrap/lib/Button';
 
 class Build extends Component {
 
-  render() {
+  getAnchorButtons() {
```

Comment:
@benrlodge do you have a convention in this project for render helpers? Internally we usually use `renderAnchorButtons`, suggesting that the helper will render some JSX.